.github/workflows/integration-tests.yml

```diff
@@ -17,8 +17,13 @@ jobs:
         database:
           - name: mysql
             pretty_name: MySQL
+            image: mysql:latest
+          - name: mysql
```

Comment:
Please have a look at https://github.com/Icinga/icingadb/blob/main/.github/workflows/sql.yml. This also avoids setting variables like `ICINGA_TESTING_PGSQL_IMAGE: mysql:latest`.

Reply:
Yes, a such env var is a bit cheaty, but I prefer a nice looking matrix with the DB type written just one (lowercase).

Reply:
> Yes, a such env var is a bit cheaty,

And setting `ICINGA_TESTING_PGSQL_IMAGE: mysql:latest` is not cheaty at all?

> but I prefer a nice looking matrix with the DB type written just one

You mean once? https://github.com/Icinga/icingadb/blob/main/.github/workflows/sql.yml doesn't repeat it any more than the current version of this PR.

> (lowercase).

That sounds just like an arbitrary choice. Sure, if both were otherwise identical, I'd also pick lowercase, but they aren't: it's easier to make the string lowercase in our [Go code](https://github.com/Icinga/icingadb/blob/89f56ac209afd7ec61afa6998f5dddaa4af1f846/tests/internal/utils/database.go#L13-L14) than in [GitHub Actions Workflow YAML](https://github.com/orgs/community/discussions/25768). And I don't think "I prefer lowercase" is a reasonable argument to completely rule out the alternative.

Reply:
No, setting such an env var IS cheaty of course (as I said), but I prefer it over

* having to lower/upper-case something somewhere
* having to repeat the database type more than once in the matrix

Reply:
> Please have a look at https://github.com/Icinga/icingadb/blob/main/.github/workflows/sql.yml. This also avoids setting variables like `ICINGA_TESTING_PGSQL_IMAGE: mysql:latest`.

I don't understand why it is not done exactly as in the referenced file.